.github/scripts/setup_generate_license.sh

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+sudo apt-get update && sudo apt-get install python3 -y
+curl https://bootstrap.pypa.io/pip/3.5/get-pip.py | sudo -H python3
+../../check_test_suite.py && travis_terminate 0 || echo 'Continuing setup'
```

Comment:
What's this reference to travis_terminate here?  I saw this in my tests too and I don't think this will work in Actions?